packages/protocol/test/identity/federatedattestations.ts

```diff
@@ -127,6 +127,33 @@ contract('FederatedAttestations', (accounts: string[]) => {
     )
   }
 
+  interface AttestationTestCase {
+    account: string
+    issuedOn: number
+    signer: string
+  }
+
+  const checkAgainstExpectedAttestations = (
+    expectedCountsPerIssuer: number[],
+    expectedAttestations: AttestationTestCase[],
+    actualCountsPerIssuer: BigNumber[],
+    actualAddresses: string[],
+    actualIssuedOns: BigNumber[],
+    actualSigners: string[]
+  ) => {
+    expect(actualCountsPerIssuer.map((count) => count.toNumber())).to.eql(expectedCountsPerIssuer)
```

Comment:
ah this one was actually from earlier, just moved it into different scope. They do the same comparison under the hood as far as I can tell (deepEqual check, checking values of properties but not that the objects are the same). Going to actually move towards assertEqualBNArray (will do the rest of the BN changes as part of [this PR](https://github.com/celo-org/celo-monorepo/pull/9633)), and will remove expect for consistency since we are using `assert.` most everywhere else!